### DISCUSSION TITLE
241220_役割分担議論MTG

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#19] [Announcements]
# 役割分担議論ミーティング 議事録

**日時**: 2023年12月19日 14:00〜14:30  
**場所**: Zoom

## 参加者
- いでっち @mikiIDE 
- ウッティ @tkg-geek 
- のり

## 議題
1. 役割分担
2. 今後の流れ

### 1. 役割分担について
- タスク洗い出し完了 by うっティ & いでちチェック（あざます！）
- [タスク分担はこちら#13](https://github.com/orgs/TeamC-by-TNI/discussions/13#discussion-7717417)
  - 1 → いでっち  
  - 4, 6 → ウッティ  
  - 2, 3, 5 → Nori  
  - 7のデザインについてはみんなで絵を描いたのをイラレでデジタル化。->うっティ

- **一番最初に終わらないとダメなのはどれ**
  - フロントはできてた方がベターだけど、データ連携などはフロントができてなくてもできる。（そんな焦らずみんなのペースで）
  - フロントのデザインの一貫性は少し注意が必要かも

### 2. 今後の流れ
- **12月21日18:30**に定例。
- 22日で各々が作れている状態を目指す。
- 23日の朝MTGで共有する。

### 3. その他
- **GitHub運用方針**:  
  - とにかく `develop` に merge していく。  
  - コードレビューはプルリクのタイミングでアナウンスがあった場合に行う。

## 決定事項
- 役割分担は決定。細かいところに気がついたらIssueを出し、みんなでフォローし合う。

## 次回MTG予定
**日時**: 2024年12月21日 18:30〜19:00  
**場所**: Zoom (NoriがSlackで共有)

**Comment by mikiIDE** [comment]
まとめ助かる…！有難うございます！
定例MTG前にできることやります！！
